src/mixin.ts

```diff
@@ -0,0 +1,21 @@
+import uuidv4 from 'uuid/v4';
+
+export default {
+  createId() {
+    let id = uuidv4();
```

Comment:
I wonder if a simple, dependency free solution can work here. Something like

```javascript
let counter = 0;

export default {
  createId() {
    let id = `block-${counter++}`;
    ...
```

Reply:
Hi, i'll use unique id's because if you have more than one instance of editor on one page, you can't find a specific block without it on DOM.